bota_ft_sensor_node:
  ros__parameters:
    serial_port: "/dev/ttyUSB0"
    frame_name: "ft_sensor_link"
    publish_intervall: 10
    zero_samples: 100
    zero_timeout_ms: 5000